### PR TITLE
8.0 oca nfe xped

### PR DESCRIPTION
**Pull request by renatonlima**
Replaces #299

### CODE REVIEW

**Comment by renatonlima** [comment]
Hello @leorochael,

Thank you for your contribution, but to we merge you PR you must sign CLA http://odoo-community.org/page/website.cla

cc @mileo

**Comment by mileo** [comment]
:+1:

**Comment by leorochael** [comment]
I remember signing the OCA CLA during Odoo Days 2015 on the OCA booth. Who should I contact to solve this?

File: l10n_br_account_product/models/account_invoice.py [hunk 1]
```diff
@@ -777,20 +777,22 @@ def _compute_price(self):
     freight_value = fields.Float(
         'Frete', digits=dp.get_precision('Account'), default=0.00)
     fiscal_comment = fields.Text(u'Observação Fiscal')
-    xped = fields.Char(
+    customer_order = fields.Char(
         string=u"Código do Pedido (xPed)",
         size=15,
     )
-    nitemped = fields.Char(
+    customer_order_line = fields.Char(
```
Comment (leorochael) [inline l10n_br_account_product/models/account_invoice.py]: @renatonlima, considering that `account.invoice` models both customers and supplier invoices, maybe we should call this field `partner_order_code` and `partner_order_line` instead?

(sorry for not replying to [your comment](https://github.com/OCA/l10n-brazil/pull/299#discussion-diff-70673545) earlier...)

**Comment by mileo** [comment]
@leorochael Can you send a email to cla AT odoo-community.org. 

Saying that you have signed in Odoo Experience, for they associate your document to login github.

**Comment by leorochael** [comment]
@mileo already did, attached with another signed CLA for good measure

**Comment by rvalyi** [comment]
@leorochael may be your CLA signature is not properly associated to your github login?
You can send a mail again to cla AT odoo-community.org to specify your github login in this case.
At Akretion, we also have folks such as @sebastienbeau at the OCA board taking care of these things but as it's 5 hours more in France right now I cannot catch him in hangout right now.

**Comment by renatonlima** [comment]
@leorochael 

Eu mudei os campos customer_order\* na fatura para partner_order*.

cc @mileo

**Comment by mileo** [comment]
@renatonlima poderia colocar o atributo oldname nos campos?

**Comment by renatonlima** [comment]
no caso é melhor colocar o xped e nitemped certo?

2016-08-15 17:13 GMT-03:00 Luis Felipe Miléo notifications@github.com:

**Comment by rvalyi** [comment]
@mileo,

Eu diria que voces poderiam fazer isso temporariamente na branch de voces para migrar os dados de voces mas isso nao e relacionado a branch da OCA. Ai se cada um comeca a lidar com as migracoes da branch temporaria dele dentro da OCA, a gente so vai encher o repo da OCA de poluicao... Aqui a gente tb as vezes tem que migrar dados de prod quando os merges acontecem, que seja na OCA ou outros repos infelizemente...

**Comment by mbcosta** [comment]
:+1:

**Comment by mileo** [comment]
With only one approval we can now merge?

**Comment by renatonlima** [comment]
@mileo you are wrong, this PR has 3 approval, (you @mbcosta and me) as you can see above this PR has 20 days open this is a refactoring from one PR made by our team #299 in 25 april I think this enough time to veto such a merge. Moreover, like a few other repos, this OCA project has few active expert reviewers so after some period of time we are forced to merge with less reviewers than projects with more expert contributors.

**Comment by rvalyi** [comment]
@mileo sorry I don't get it. You initially approved the PR and then wanted to add old_name field attributes to help to migrate your own branch data. Sorry but no OCA project assume to help the PR authors to migrate their own production data when refactors are required. Yes it sucks, but frankly at Akretion we migrate our prototype branch data at customers all the time due to required refactors before our PR enters the OCA, it just happens to us all the time... We are about to invite you to the project PSC once we finish an urgent structuring PR but frankly we still have some doubts when you want to merge that kind of things or when you wanted to merge the pysped dependent modules here or your non OCA compliant 8.0 migration attempt branch...

**Pull request merged**